src/main/java/uk/gov/hmcts/divorce/divorcecase/tab/CaseTypeTab.java

```diff
@@ -312,6 +313,13 @@ private void buildGeneralReferralTab(ConfigBuilder<CaseData, State, UserRole> co
             .field("generalReferrals");
     }
 
+    private void buildGeneralApplicationTab(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        configBuilder.tab("generalApplication", "General Application")
```

Comment:
Thinking about whether we should change the name to General Applications now it shows all the general apps rather than the most recent?